package.json

```diff
@@ -77,6 +77,7 @@
     "glider-js": "^1.7.6",
     "history": "^5.0.0",
     "html-to-image": "github:merapi/html-to-image",
+    "lodash": "^4.17.21",
```

Comment:
Important to note that we're not planning on using lodash anywhere in our application.

This is just for the block and lodash is loaded separately in WordPress and always available.